src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement, useState } from 'react'
+import { TAvatarProps } from './types'
+import classNames from 'classnames'
+import FileUpload from '../FileUpload'
+import { FileUploadMode } from '../FileUpload/types'
+import '../../assets/styles/components/_avatar.scss'
+
+export const Avatar = ({
+  id,
+  color = 'default',
+  size = 'medium',
+  imagePath = '',
+  initials = '',
+  className = '',
+  isEditable = false,
+  onAvatarChange
+}: TAvatarProps): ReactElement => {
+  const [image, setImage] = useState<string>(imagePath)
+  const getFiles = (files: File[]) => {
+    if (files && files[0]) {
+      setImage(URL.createObjectURL(files[0]))
+      onAvatarChange?.(files[0])
+    }
+  }
+  const style = {
+    ...(image ? { backgroundImage: image ? `url(${image})` : 'none' } : null)
```

Comment:
you are checking image twice . `{ backgroundImage: image ? `url(${image})` : 'none' }` here it's already exist